website/client/store/actions/auth.js

```diff
@@ -65,8 +65,8 @@ export async function socialAuth (store, params) {
   localStorage.setItem(LOCALSTORAGE_AUTH_KEY, userLocalData);
 }
 
-export function logout () {
+export async function logout () {
   localStorage.removeItem(LOCALSTORAGE_AUTH_KEY);
   localStorage.removeItem(LOCALSTORAGE_SOCIAL_AUTH_KEY);
-  window.location.href = '/logout';
+  await axios.get('/logoutServer');
```

Comment:
this shouldn't also redirect the user to the homepage? So instead of doing axios.get why not use `window.location.href = '/logoutServer';	 `?